3rdparty/build_aubio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo export CFLAGS=${3}
+export CFLAGS=${3}
+
+echo mkdir -p $2 && mkdir -p $2
+
+echo "${BASH_SOURCE%/*}/aubio/waf" --verbose configure build install $1 --enable-fftw3f --disable-sndfile --disable-avcodec --disable-double --disable-samplerate --disable-docs --disable-wavread --disable-wavwrite --disable-tests --notests --disable-examples --disable-apple-audio --disable-fat --disable-jack --prefix=$2 --out=$2 --top="${BASH_SOURCE%/*}/aubio/"
```

Comment:
Wrap this up in a variabele and use it to echo and than run it so its defined once.
Maybe configure this to build with cmake instead of a custom script? Atm i dont have a clue how though

Reply:
It IS configured to build via a CMake script; the script calls bash here because else I couldn't get it to properly build out of source.